specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/examples/NorthSouthHardenings/ListNorthSouthHardenings_example.json

```diff
@@ -0,0 +1,50 @@
+{
+    "parameters": {
+        "api-version": "2015-06-01-preview",
+        "subscriptionId": "20ff7fc3-e762-44dd-bd96-b71116dcdc23"
+    },
+    "responses": {
+        "200": {
+            "body": {
+                "value": [
+                    {
+                        "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/resourcegroups/rg1/providers/Microsoft.Security/northSouthHardenings/vm1",
+                        "name": "vm1",
+                        "type": "Microsoft.Security/northSouthHardenings",
+                        "properties": {
+                            "protectedResourceId": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/resourcegroups/rg1/providers/Microsoft.Compute/virtualMachines/vm1",
+                            "policies": [
+                                {
+                                    "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/resourcegroups/rg1/providers/Microsoft.Security/northSouthHardening/vm1/policies/recommended",

```

Comment:
what are the possible names for policy? can the user give any name when creating? use of "default" in the name makes me want to double check.

Reply:
We create a default policy in the background and its name will be "recommended".
A user can create or update other policies but not the "recommended" one